app/Controllers/Http/AuthController.ts

```diff
@@ -0,0 +1,41 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import User from 'App/Models/User'
+import Profile from 'App/Models/Profile'
+import RegisterValidator from 'App/validators/RegisterValidator'
+
+export default class AuthController {
+  public async register({ request, response }: HttpContextContract) {
+    const { email, password, mobile, dob, gender } = await request.validate(RegisterValidator)
```

Comment:
```suggestion
    const { email, password, mobile, dob, gender, name } = await request.validate(RegisterValidator)
```

Reply:
@sombirCredilio  do not mark the comments as resolved. Just reply to it that it it fixed or resolved